src/wallet/wallet2.cpp

```diff
@@ -4700,6 +4896,7 @@ bool wallet2::load_keys_buf(const std::string& keys_buf, const epee::wipeable_st
     m_ignore_outputs_above = MONEY_SUPPLY;
     m_ignore_outputs_below = 0;
     m_track_uses = false;
+    m_background_sync_type = BackgroundSyncOff;
```

Comment:
should we also set `m_is_background_wallet` here?

Reply:
It's set above already